test/private/execute_hash_stage.m

```diff
@@ -28,7 +28,7 @@
     % By storing the hash table in a persistent variable, the amount of disk
     % operations is minimized (i.e. reading the file, parsing it and storing its
     % data in a MATLAB struct), as this is only done once a new test suite is
-    % executed. To clear this persistent storage, run |clear functions|.
+    % executed. To clear this persistent storage, run `clear getReferenceHash`.
```

Comment:
I wasn't aware that this is MATLAB markup -- never have seen it before and thus changed it. Thus I certainly don't have hard feelings about this being changed to something else.